docs/renovate.md

```diff
@@ -0,0 +1,46 @@
+# Handling Renovate PRs
+We use [Renovate](https://github.com/renovatebot/renovate) to monitor our PHP and JavaScript package dependencies.
+Renovate will open a PR when an update is available for a package. We can then review the PR, test the change, and
+merge to apply the update. The testing and review needed will depend on the package being updated. This document aims
+to catalog our packages and provide guidance to a developer who wants to test and merge Renovate PRs.
+
+## Some Notes on Packages
+* Sometimes Renovate creates a PR for a bundle of packages and calls it a monorepo update. This is default Renovate
+  behaviour for when it thinks a package belongs to a GitHub repository using something like Lerna to adminster multiple
+  packages in a single repository. This behaviour makes sense for us and helps reduce noise from Renovate PRs.
+* A lot of our JavaScript dev dependencies are provided by WordPress or WooCommerce globally at runtime, we include them
+  as dev dependencies so that we aren't duplicating them in our build bundle but our unit tests can still pass by
+  having them available. This means we should keep the versions of these packages in the same range as our supported 
+  version of WordPress and WooCommerce.
```

Comment:
That's great point, however, I was always struggling with finding the supported versions range for specific dependency packages. E.g. which versions of  wordpress/components we should support/use in tests?

@jrodger do you know a good way of checking those?